src/main/drivers/flash_impl.h

```diff
@@ -32,20 +56,30 @@ struct flashVTable_s;
 typedef enum {
     FLASHIO_NONE = 0,
     FLASHIO_SPI,
-    FLASHIO_QUADSPI
+    FLASHIO_QUADSPI,
+    FLASHIO_OCTOSPI,
 } flashDeviceIoMode_e;
 
 typedef struct flashDeviceIO_s {
     union {
+    #ifdef USE_FLASH_SPI
         extDevice_t *dev; // Device interface dependent handle (spi/i2c)
```

Comment:
no I2C now ?

Reply:
this PR hasn't changed this, see the initial QUADSPI commit (dcd138ae20672a3305dcf3aa1a19cab8c955dbad) from a few years ago which added `FLASHIO_SPI`.  There's never been `FLASHIO_I2C`.  If flash by I2C support is added then the PR to add it can address this.